email/1.3/modules/ROOT/edits/email-studio-configuration.adoc

```diff
@@ -0,0 +1,147 @@
+
+If you are new to configuring connectors in Studio, see xref:connectors::introduction/intro-config-use-studio.adoc[Use Studio to Configure a Connector]. If, after reading this topic, you need additional information about the connector fields, see the xref:email-documentation.adoc[Email Connector Reference].
+
+[[create-mule-project]]
+== Create a Mule Project
+
+In Studio, create a new Mule project in which to add and configure the connector:
+
+. In Studio, select *File > New > Mule Project*.
+. Enter a name for your Mule project and click *Finish*.
+
+
+[[add-connector-to-project]]
+== Add the Connector to Your Mule Project
+
+Add Email Connector to your Mule project to automatically populate the XML code with the connector's namespace and schema location and to add the required dependencies to the project's `pom.xml` file:
+
+. In the *Mule Palette* view, click *(X) Search in Exchange*.
+. In the *Add Dependencies to Project* window, type `email` in the search field.
+. Click *Email Connector* in *Available modules*.
+. Click *Add*.
+. Click *Finish*.
+
+Adding a connector to a Mule project in Studio does not make that connector available to other projects in your Studio workspace.
+
+[[configure-input-source]]
+== Configure a Source
+
+A source initiates a flow when a specified condition is met.
+You can configure one of these input sources to use with Email Connector:
+
+* *On New Email - IMAP* +
+Initiates a flow by retrieving all the emails from an IMAP mailbox folder, watermark can be enabled for polled items
+* *On New Email - POP3* +
+Initiates a flow by retrieving all the emails from an POP3 mailbox folder
+* *HTTP > Listener* +
+Initiates a flow each time it receives a request on the configured host and port
+* *Scheduler* +
+Initiates a flow when a time-based condition is met
+
+For example, to configure the *On New Email - IMAP* source , follow these steps:
+
+. In the *Mule Palette* view, select *Email > On New Email - IMAP*.
+. Drag *On New Email - IMAP* to the Studio canvas.
+. On the *On New Email - IMAP* configuration screen, optionally change the value of the *Display Name* field.
+. Click the plus sign (*+*) next to the *Connector configuration* field to configure a global element that can be used by all instances of the source in the app.
+. On the *Email IMAP* window, for *Connection* select any of the connection types to provide to this configuration:
+
+* *IMAP Connection*
+* *IMAPS Connection*
+
+[start=6]
+. On the *General* tab, specify the connection information for the connector, such as *Host*, *Port* and *TLS configuration*.
+. On the *Advanced* tab, optionally specify timeout configuration and reconnection information, including a reconnection strategy.
+. Click *OK*.
+. On the *On New Email - IMAP* configuration screen, in the *General* section, select a *Scheduling Strategy* to configure the scheduler that triggers the polling.
```

Comment:
```suggestion
. In the *On New Email - IMAP* configuration screen, in the *General* section, select a *Scheduling Strategy* to configure the scheduler that triggers the polling.
```

Reply:
Does the template use "on a window," "on a tab," and "on a screen"?
Seeing it here sounds wrong. To me, "on" = on top of, and "in" = inside (an area).
Is this something the team should discuss? Have I just let this go in the past?